.github/workflows/build-test.yml

```diff
@@ -60,15 +70,7 @@ jobs:
       - name: Work around git safe.directory in container
         run: chown -R $(id -u):$(id -g) $PWD
 
-      # Manually disable Launchpad repository to reduce the chances of a supply chain attack
-      - name: Disable Launchpad and PPA repositories
-        run: |
-          sudo sed -i '/launchpad.net/d' /etc/apt/sources.list
-          sudo sed -i '/ppa.launchpad.net/d' /etc/apt/sources.list
-          sudo find /etc/apt/sources.list.d/ -type f -exec sed -i '/launchpad.net/d' {} \;
-          sudo find /etc/apt/sources.list.d/ -type f -exec sed -i '/ppa.launchpad.net/d' {} \;
```

Comment:
Is it not an issue anymore, previously those were part of the image

Reply:
Launchpad and PPA repositories were used for Ubuntu / Debian packages. This should no longer be a problem since Azure Linux 3 uses RPM as a package manager.